_partners/spot.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Spot by NetApp
+# upload your logo to the following directory - must be square
+logo: '/assets/media/partners/spot-by-netapp.png'
```

Comment:
These filenames are case sensitive - it looks like you've uploaded 'assets/media/partners/Spot-by-NetApp.png` but the logo front matter here refers to `spot-by-netapp.png`

Everything else otherwise looks great!

Reply:
If you're able to run `bundle exec jekyll serve` from the cli, you can run your own copy of the site locally and visit http://localhost:4000/ to see what your page looks like. It's a handy debugging step.